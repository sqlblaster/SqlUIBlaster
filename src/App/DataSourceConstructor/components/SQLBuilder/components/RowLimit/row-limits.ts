export const rowLimits = [null, 1, 10, 25, 100, 1000, 2000];
