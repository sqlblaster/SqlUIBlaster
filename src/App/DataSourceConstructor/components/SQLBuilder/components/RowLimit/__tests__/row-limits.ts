import { rowLimits } from '../row-limits';

describe('row limits list', () => {
  it('should match snapshot', () => {
    expect(rowLimits).toMatchSnapshot();
  });
});
