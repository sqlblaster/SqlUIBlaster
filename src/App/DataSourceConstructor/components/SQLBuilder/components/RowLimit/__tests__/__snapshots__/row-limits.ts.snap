// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`row limits list should match snapshot 1`] = `
Array [
  null,
  1,
  10,
  25,
  100,
  1000,
  2000,
]
`;
